### ISSUE TITLE
Android Support

### ISSUE THREAD

**Issue by rebeccahughes** [jsdf/react-native-refreshable-listview#23]
Are there any plans to add android support?

**Comment by icefoggy** [comment]
+1 As ReactNative for Android has been released...

**Comment by ajdevy** [comment]
+1 , struggling on the project

**Comment by kinhunt** [comment]
+1

**Comment by buhe** [comment]
+1 ..

**Comment by han4wluc** [comment]
＋1

**Comment by jsdf** [comment]
Yes, there are plans. I don't know when I'll have time to implement it, however.

**Comment by bnolan** [comment]
What ios components does this use?

**Comment by bnolan** [comment]
Looks like it just uses ActivityIndicatorIOS. Should be an easy fix?

**Comment by buhe** [comment]
android not support pull listview..

**Comment by PhilippKrone** [comment]
@jsdf Android support would be highly appreciated - can I support in any way here?

**Comment by southerneer** [comment]
+1

**Comment by maraujop** [comment]
It's my understanding that for getting Android support we should simply fix this api inconsistency https://github.com/facebook/react-native/issues/3725

**Comment by misakuo** [comment]
+1

**Comment by PhilippKrone** [comment]
@maraujop Indeed yeah. Either implement the API as mentioned in https://github.com/facebook/react-native/issues/3725 or wait until SwipeRefreshLayout is open sourced and use it in react-native-refreshable-listview.

**Comment by sonnylazuardi** [comment]
+1

**Comment by richarddewit** [comment]
+1

**Comment by changfuguo** [comment]
hi ，here i  made a demo , but it just like pulldown,  but i think if we just complete the work by js,, it is a way to realize it , the article  is here  [react－native-pulldown-listview](https://github.com/changfuguo/react-native/blob/master/listview-pulldown.md)

 ![](https://raw.githubusercontent.com/changfuguo/doukanmv/master/temp/pulldown.gif)

**Comment by aug-riedinger** [comment]
+1 of course

**Comment by richarddewit** [comment]
In [RN v0.16.0](https://github.com/facebook/react-native/releases/tag/v0.16.0), `PullToRefreshLayoutAndroid` is implemented. ([Example](https://github.com/facebook/react-native/blob/37f81341a006458d84d934882485c2a0a8838497/Examples/UIExplorer/PullToRefreshLayoutAndroidExample.android.js))

Would be nice to put it in here.

**Comment by niftylettuce** [comment]
:+1: Yeah definitely!

**Comment by pnpnpn** [comment]
+1

**Comment by kaminskypavel** [comment]
any plans to integrate with https://facebook.github.io/react-native/docs/pulltorefreshviewandroid.html#content
?

**Comment by jsdf** [comment]
@kaminskypavel would be good to make use of. Don't have much time to explore it at the moment.

**Comment by maraujop** [comment]
I've just submitted a PR adding support for Android

**Comment by kaminskypavel** [comment]
gotta love the power of community #41
